opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -498,6 +526,7 @@ export const addUser = async (context, user, newUser) => {
     R.assoc('account_status', newUser.account_status ? newUser.account_status : DEFAULT_ACCOUNT_STATUS),
     R.assoc('account_lock_after_date', newUser.account_lock_after_date),
     R.assoc('unit_system', newUser.unit_system),
+    R.assoc('user_confidence_level', { max_confidence: nconf.get('app:user_confidence_level_default') ?? 100, overrides: [] }),
```

Comment:
user creation does not take the confidence as input; it's an internal attribute.
We will be able to update the confidence once created ofc.

Reply:
maybe `nconf.get('app:user_confidence_level_default')` should be parsed in integer before setting it here ? and fallback to 100 in any case ?

Reply:
I just checked, and nconf returns it as a number, no need for parsing.